packages/websocket-plugin/src/symbols.ts

```diff
@@ -74,6 +74,16 @@ export class DisconnectWebSocket {
   }
 }
 
+/**
+ * Action triggered when websocket is disconnected
+ */
+export class WebSocketDisconnected {
+  static get type() {
+    // NOTE: Not necessary to declare the type in this way in your code. See https://github.com/ngxs/store/pull/644#issuecomment-436003138
+    return '[Websocket] Disconnected';
```

Comment:
All the scopes should be with a capital S to match all the other stuff that is written WebSocket.

Reply:
@eranshmil
rename for consistency with little S everywhere